README.md

```diff
@@ -31,6 +31,27 @@ cache works between processes)
 
 ## Usage
 
+Add it in your root build.gradle at the end of repositories:
+
+```groovy
+
+allprojects {
+  repositories {
+    ...
+    maven { url "https://jitpack.io" }
+  }
+}
+```
+	 
+Add the dependency
+
+```groovy
+
+dependencies {
+   compile "com.github.iamironz:binaryprefs:1.0.0-ALPHA-1"
```

Comment:
I think you can grab the badge from jitpack as markdown: `[![](https://jitpack.io/v/iamironz/binaryprefs.svg)](https://jitpack.io/#iamironz/binaryprefs)` and place it to last badges block right in top of readme file. 
Also instead of using hardcoded version here just write `compile "com.github.iamironz:binaryprefs:$version"`